### PR TITLE
✈️ add public app

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

This PR adds a public app for the combined somatic workflow. 

Closes https://github.com/d3b-center/bixu-tracker/issues/765

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- [x] cwltool validation
- [x] Cavatica test: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/30f14232-1245-4ae3-9b4e-8220d4e29f42/#

**Test Configuration**:
* Environment: cwltool 3.0.20200324120055, cavatica
* Test files: see cavatica test

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: workflow/kfdrc-somatic-variant-workflow.cwl [hunk 1]
```diff
@@ -0,0 +1,800 @@
+cwlVersion: v1.0
+class: Workflow
+id: kfdrc-somatic-variant-workflow
+label: Kids First DRC Somatic Variant Workflow
+doc: |
+  # KFDRC Somatic Whole Genome Sequence Analysis Workflow
+
+  ![data service logo](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS9BnbvIsTkK3QlSGMDvlgu0tZQJ1q4crMvA-S3fcWfIq6y2d2Y)
+
+  This is the Kids First Data Resource Center (DRC) Whole Genome Sequencing (WGS) Somatic Workflow, which includes somatic variant calling, copy number variation (CNV), and structural variant (SV) calls.
```
Comment (migbro) [inline workflow/kfdrc-somatic-variant-workflow.cwl:10]: Same comment as before for this and readme, just slide it above the logo

Comment (dmiller15) [reply workflow/kfdrc-somatic-variant-workflow.cwl]: Say no more.

**Comment by dmiller15** [comment]
Ok @migbro I did a little consolidation for the readme and used that to update the doc. Should be a little less wordy now.

**Review by sickler-alex — APPROVED** [review]

**Review by migbro — CHANGES_REQUESTED** [review]
Minor request

**Review by migbro — APPROVED** [review]

**Pull request merged**